pkg/replica/backup.go

```diff
@@ -154,24 +158,33 @@ type BackupStatus struct {
 	BackupURL     string
 	State         ProgressState
 	IsIncremental bool
+	IsOpened      bool
 }
 
-func NewBackup(backingFile *backingfile.BackingFile) *BackupStatus {
+func NewBackup(name, volumeID, snapID string, backingFile *backingfile.BackingFile) *BackupStatus {
+	backupName := name
+	if backupName == "" {
+		backupName = butil.GenerateName("backup")
+	}
 	return &BackupStatus{
+		Name:        backupName,
 		backingFile: backingFile,
-		State:       ProgressStateInProgress,
+		State:       ProgressStateStarting,
+		volumeID:    volumeID,
+		SnapshotID:  diskutil.GenerateSnapshotDiskName(snapID),
```

Comment:
Why not just use the original `snapID`?

Reply:
`GenerateSnapshotDiskName` adds the prefix `volume-snap-` to snapID.